R/redist_constr.R

```diff
@@ -265,6 +294,33 @@ add_constr_grp_hinge = function(constr, strength, group_pop, total_pop=NULL,
     add_to_constr(constr, "grp_hinge", new_constr)
 }
 
+
+#' @param tgts_group A vector of target group shares for the hinge-type constraint.
+#' @rdname constraints
+#' @export
+add_constr_grp_iv_hinge = function(constr, strength, group_pop, total_pop=NULL,
```

Comment:
typo `iv` -> `inv`

what do you think about just adding an `inverse=FALSE/TRUE` flag to `grp_hinge`?

Reply:
I think I'm a fan of separate functions, given the way the constraints read out when you pipe them. I think that is easier to work with, especially if it doesn't add much to maintain